File: android/app/src/main/java/com/vegettable/app/model/Market.java
package com.vegettable.app.model;

import com.google.gson.annotations.SerializedName;

public class Market {
    @SerializedName("marketCode")
    private String marketCode;

    @SerializedName("marketName")
    private String marketName;

    @SerializedName("region")
    private String region;

    public String getMarketCode() { return marketCode; }
    public String getMarketName() { return marketName; }
    public String getRegion() { return region; }
}

File: android/app/src/main/java/com/vegettable/app/model/Market.kt
package com.vegettable.app.model

import com.google.gson.annotations.SerializedName

class Market {
    @SerializedName("marketCode")
    val marketCode: String? = null

    @SerializedName("marketName")
    val marketName: String? = null

    @SerializedName("region")
    val region: String? = null

    @SerializedName("address")
    val address: String? = null

    @SerializedName("latitude")
    val latitude: Double = 0.0

    @SerializedName("longitude")
    val longitude: Double = 0.0
}
